docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -94,61 +94,90 @@ expands on what differentiates IBM, including microsimulation, from other types
+Simulation methods have the added advantage of reducing confounding variance between different scenarios by ensuring that the simulated population is identical across all scenarios analyzed. 
+The only difference between scenarios, then, is the presence or absence of health interventions. By incorporating heterogeneity at the level of the individual simulant, microsimulations are 
+able to provide an additional degree of flexibility and detail (such as stratifying results according to demographic or epidemiological attributes) not available in population-level models. 
 
-- Surfaces complications that are more hidden in non-individual-based models (e.g., compartmental models can obscure certain limitations)
+Allowing other complexities
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Disadvantages of microsimulation
-++++++++++++++++++++++++++++++++
+Another benefit of microsimulations in general is that they can surface complications or limitations that are obscured in non-individual-based models, such as compartmental models.
+
+.. todo:: 
+  
+  Elaborate on example of this from previous work: examining hemoglobin on curve on an individual level and seeing the discrepancies from real life that would also be present in a 
+  model that doesn't go to the level of detail we do in microsim. 
 
-.. todo::
 
-  Fill out the following sections under 'Disadvantages of microsimulation' subheading.
+Disadvantages of microsimulation
+++++++++++++++++++++++++++++++++
 
 Resource requirements
 ~~~~~~~~~~~~~~~~~~~~~
 
-Our microsimulation models require a significant amount of detailed data to accurately represent individual behaviors and the relationships between them. Obtaining this data can be challenging and time-consuming, and there may be privacy concerns associated with using such detailed personal data. 
-
-If we're not modeling the sort of complexity that microsimulation enables... perhaps a multiplication model would suit us just as well?
+Our microsimulation models require a significant amount of detailed data to accurately represent individual behaviors and the relationships between them. 
```

Comment:
I think of this as "data requirements," and as almost exactly the flip side of the "allowing other complexities" coin. So, I think we should either explicitly link the two, and list it as both an advantage and a disadvantage, or lean into being salespeople and remove this disadvantage, leaving only the advantage 😄

However, there is a disadvantage I would call "resource requirements," and that is that our simulations take a lot of _computational_ resources to run. That should definitely be listed here.

Reply:
I agree about the "two sides of the same coin" part and connecting them would be cool! But I do think this is important to include. We have definitely run into issues with CVD where the data on how many people take their medications, or go to the doctor, or restart meds after discontinuing is really hard to find.